actions/reservation_unit.py

```diff
@@ -0,0 +1,105 @@
+from typing import Any
+
+from opening_hours.errors import HaukiAPIError, HaukiRequestError, HaukiValueError
+from opening_hours.models import OriginHaukiResource
+from opening_hours.utils.hauki_api_client import HaukiAPIClient
+from opening_hours.utils.hauki_api_types import HaukiAPIResource, HaukiTranslatedField
+from reservation_units.models import ReservationUnit
+
+
+class ReservationUnitHaukiExporter:
```

Comment:
Could rename this to `ReservationUnitActions`, create a connector in `/common/connectors.py` and then add that to the ReservationUnit model. Then, in `/api/graphql/types/reservation_units/mutations.py:ReservationUnitMutationMixin` instead of:

```python
exporter = ReservationUnitHaukiExporter(reservation_unit)
try:
    exporter.send_reservation_unit_to_hauki()
except (HaukiRequestError, HaukiAPIError):
    ...
```

...you could do:

```python
try:
    reservation_unit.actions.send_reservation_unit_to_hauki()
except (HaukiRequestError, HaukiAPIError):
    ...
```